app/hooks/useCommandBarActions.ts

```diff
@@ -11,7 +11,10 @@ import useActionContext from "./useActionContext";
  *
  * @param actions actions to make available
  */
-export default function useCommandBarActions(actions: Action[]) {
+export default function useCommandBarActions(
+  actions: Action[],
+  additionalDeps: string[] = []
```

Comment:
```suggestion
  additionalDeps: React.DependencyList = []
```

No functional difference, but it's nice